src/ert/gui/model/snapshot.py

```diff
@@ -43,21 +42,17 @@
 
 DURATION = "Duration"
 
-COLUMNS: Dict[NodeType, Sequence[str]] = {
-    NodeType.ROOT: ["Name", "Status"],
-    NodeType.ITER: ["Name", "Status", "Active"],
-    NodeType.REAL: [
-        ids.NAME,
-        ids.ERROR,
-        ids.STATUS,
-        DURATION,  # Duration is based on two data fields, not coming directly from ert
-        ids.STDOUT,
-        ids.STDERR,
-        ids.CURRENT_MEMORY_USAGE,
-        ids.MAX_MEMORY_USAGE,
-    ],
-    NodeType.JOB: [],
-}
+JOB_COLUMNS: Sequence[str] = [
+    ids.NAME,
+    ids.ERROR,
+    ids.STATUS,
+    DURATION,  # Duration is based on two data fields, not coming directly from ert
+    ids.STDOUT,
+    ids.STDERR,
+    ids.CURRENT_MEMORY_USAGE,
+    ids.MAX_MEMORY_USAGE,
```

Comment:
Is it possible to remove all the constants from identifiers (ids) and have them here instead?

Reply:
Not easily no. A lot of these identifiers are used elsewhere. I suspect the net gain is negative from such an impact.